--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -35,6 +35,7 @@ execute_process(
     ${Python_EXECUTABLE}
     dump_symbols.py
     "${occt_libs}"
+    WORKING_DIRECTORY ${CMAKE_SOURCE_DIR}
     )
 
 
@@ -71,7 +72,6 @@ if( NOT EXISTS ${CMAKE_SOURCE_DIR}/OCP )
             -l ${LIBCLANG_PATH}
             -i ${CLANG_INCLUDE_DIRS}/
             -i ${VTK_INCLUDE_DIR}/
-            -i ${CLANG_INSTALL_PREFIX}/lib/clang/${LLVM_VERSION}/include/
             ${CXX_INCLUDES}
             all ocp.toml ${PLATFORM}
         WORKING_DIRECTORY ${CMAKE_SOURCE_DIR}
